frontend/src/BenchmarkTest.res

```diff
@@ -36,10 +95,13 @@ let decodeMetricValue = (json): LineGraph.DataRow.value => {
 }
 
 let decodeMetric = (data): LineGraph.DataRow.metric => {
+    let name = (Js.Dict.get(data, "name")->Belt.Option.getExn->Js.Json.decodeString->Belt.Option.getExn)
+    let units_ = (Js.Dict.get(data, "units")->Belt.Option.getExn->Js.Json.decodeString->Belt.Option.getExn)
+    let (value, units) = decodeMetricValue(Js.Dict.get(data, "value")->Belt.Option.getExn, units_)
```

Comment:
If the units are different between two different values displayed in the graph, the graph doesn't make sense since we only show the units for the latest observation in the UI. The value displayed and the value used in the graph should probably be different? Should all the values in the graph be converted to the units of the last observation? :thinking:

Reply:
I'm not sure what the exact issue is I'm not sure how the UI should be reflected, can you elaborate more, Thanks!

Reply:
![image](https://user-images.githubusercontent.com/315678/141300815-330a5a5e-9704-4363-9f78-950ddedc9d45.png)

So, the first value here in the graph was mbs, and the other values were many times larger than the previous values. But, since we change the value and the units for each observation independently, the graph doesn't look right. 

Values like 200mb, 2000mb, 2_000_000mb, etc could make the graph look like the above.

Reply:
Ooh this is a good point @punchagan , in practice it doesn't really happen (unless someone changes the benchmarks dramatically!). But perhaps we can handle it, something to do can be to convert the last x values (that are displayed on the graph) also.

Reply:
I understand that my example is probably too extreme, but it could happen in some edge cases where the value changes from 999 bytes to 1001 bytes. With the current code, the second value gets plotted as 1.001 kb, which would make the graph wrong. Yes, converting the last few values being displayed on the graph seems like a reasonable thing to do.

Reply:
Ooh yes, great point!

Reply:
Thanks for clarifying! This is a great point, the only solution that I can think to solve this issue is to normalize the display data because I don't know how to access the displayed value specifically in the graph :)